src/redux/reducers/tweets.ts

```diff
@@ -151,14 +152,21 @@ const tweetsSlice = createSlice<TweetsStore, SliceCaseReducers<TweetsStore>>({
     builder.addCase(fetchFeedTweets.pending, (store) => {
       store.feedTweets.isLoading = true;
     });
+
     builder.addCase(fetchFeedTweets.fulfilled, (store, { payload }) => {
       if (store.feedTweets.docs.length === 0 || payload.page !== 1) {
         // I don't know why, but first page we have twice
         // and only such way I resolved this problem
         store.feedTweets = {
           ...store.feedTweets,
           ...payload,
-          docs: [...store.feedTweets.docs, ...payload.docs],
+          docs: [
+            ...store.feedTweets.docs,
+            ...payload.docs.map((item) => ({
+              ...item,
+              createdAt: localDateTime(item.createdAt),
```

Comment:
it's incorrect way
you shouldn't change any data, that you received from backend before setting it to store. Redux store, it's kinda your backend DB on frontend. Instead, you should make formatting in render

Reply:
fixed